### PR TITLE
analysis: which are the biggest officer clusters/cliques and with which percentage are swiss officers a part of them

### LINKED ISSUE #7: analysis: which are the biggest officer clusters/cliques and with which percentage are swiss officers a part of them

### PR DESCRIPTION
**Pull request by florinbarbisch**

### CODE REVIEW

File: 99-Swiss officer ratios.ipynb [hunk 1]
```diff
@@ -0,0 +1,959 @@
+{
```
Comment (buesst1) [inline 99-Swiss officer ratios.ipynb]: <p>nx.projected_graph ;) but ok</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gabrieltorresgamez/fhnw-san-project/pull/13/discussion/#comment-1609516190'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='7'/>

Comment (buesst1) [inline 99-Swiss officer ratios.ipynb]: <p>Please add a couple comments to the code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gabrieltorresgamez/fhnw-san-project/pull/13/discussion/#comment-1609516193'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='10'/>

Comment (buesst1) [inline 99-Swiss officer ratios.ipynb]: <p>Title is a bit confusing.</p><p>Number of unique ratios, first five ratios -&gt; textbox in plot (please round numbers)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gabrieltorresgamez/fhnw-san-project/pull/13/discussion/#comment-1609516196'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='11'/>

Comment (buesst1) [inline 99-Swiss officer ratios.ipynb]: <pre class="ql-syntax" spellcheck="false">Line #2.    plot_swiss_officer_ratios_by_group(new_officers_graph, lambda G: k_clique_communities(G, n), f'{n}-clique')
</pre><p>I dont know what this function does but I think its not equivalent to n-cliques in the san script.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gabrieltorresgamez/fhnw-san-project/pull/13/discussion/#comment-1609516199'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='15'/>

**Review by buesst1 — CHANGES_REQUESTED** [review]
pliiz rename notebook to 02....

See comments above

File: 99-Swiss officer ratios.ipynb (hunk 1 above)
Comment (florinbarbisch) [reply 99-Swiss officer ratios.ipynb]: yes, I declared that now

**Review by buesst1 — APPROVED** [review]
LGTM

**Pull request merged**